sites/github.styl

```diff
@@ -12,8 +12,8 @@
 
 // ** Misc
 
+.link-gray-dark
 a.link-gray-dark
```

Comment:
Since you added the plain class selector, can we remove the `a.link-gray-dark` selector?